core/deployment/src/main/java/io/quarkus/deployment/steps/ReflectiveHierarchyStep.java

```diff
@@ -36,23 +34,17 @@
 
     private static final Logger log = Logger.getLogger(ReflectiveHierarchyStep.class);
 
-    @Inject
-    List<ReflectiveHierarchyBuildItem> hierarchy;
-
-    @Inject
-    CombinedIndexBuildItem combinedIndexBuildItem;
-
-    @Inject
-    BuildProducer<ReflectiveClassBuildItem> reflectiveClass;
-
-    @Inject
-    List<ReflectiveHierarchyIgnoreWarningBuildItem> ignored;
-
-    @Inject
-    List<ReflectiveClassFinalFieldsWritablePredicateBuildItem> finalFieldsWritablePredicates;
```

Comment:
Please do that in a separate commit next time. It makes the review far harder than it needs to be.

Reply:
You are absolutely right. I thought of it myself, but I guess I decided against it because I (stupidly) decided it would be best to economize on commits :laughing: